ansible/roles/ocp-workload-iot-demo/files/certs/readme.md

```diff
@@ -0,0 +1,3 @@
+# Warning!
+The certs in this directory are self generated placeholders for the initial installation. It is expected that 
+these will be replaced soon after installation by updating the ec_cert secret in the Openshift project.
```

Comment:
it seems always strange to have cert keys versioned in the code. Isn't it possible to generate cert and key before the installation instead ?

Reply:
@fridim I factored this out and borrowed some code from another similar role. everything under /certs has been removed and is generated now.